app/utils/transformer/DepartureDataTransformer.scala

```diff
@@ -52,7 +53,22 @@ class DepartureDataTransformer @Inject() (
   representativePhoneNumberTransformer: RepresentativePhoneNumberTransformer,
   containerIndicatorTransformer: ContainerIndicatorTransformer,
   modeOfTransportAtTheBorderTransformer: ModeOfTransportAtTheBorderTransformer,
-  addBorderModeOfTransportYesNoTransformer: AddBorderModeOfTransportYesNoTransformer
+  addBorderModeOfTransportYesNoTransformer: AddBorderModeOfTransportYesNoTransformer,
+  addContactYesNoTransformer: AddContactYesNoTransformer,
+  addIdentifierYesNoTransformer: AddIdentifierYesNoTransformer,
+  additionalIdentifierTransformer: AdditionalIdentifierTransformer,
+  addressTransformer: AddressTransformer,
+  authorisationNumberTransformer: AuthorisationNumberTransformer,
+  coordinatesTransformer: CoordinatesTransformer,
+  countryTransformer: CountryTransformer,
+  customsOfficeIdentifierTransformer: CustomsOfficeIdentifierTransformer,
+  eoriTransformer: EoriTransformer,
+  locationOfGoodsIdentificationTransformer: LocationIdentificationTransformer,
+  locationTypeTransformer: LocationTypeTransformer,
+  nameTransformer: NameTransformer,
+  phoneNumberTransformer: PhoneNumberTransformer,
+  postalCodeTransformer: PostalCodeTransformer,
+  unLocodeTransformer: UnLocodeTransformer
```

Comment:
This might just be a personal preference but this feels like a lot of dependencies for one class. Could we perhaps create a transformer class for each val that we have below?